src/lesson8/Task6.java

```diff
@@ -0,0 +1,34 @@
+package lesson8;
+
+/*6. Write a program that will read numbers from the keyboard.
+        The code for reading numbers from the keyboard must be in the readData method.
+        The code inside the readData is wrapped in a try..catch.
+        If the user entered some text, instead of a number, this method should catch the exception and display all
+        previously entered numbers as a result. Each number should be printed from a new line
+        preserving the order of the input.*/
+
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Task6 {
+    public static void main(String[] args) {
+        readData();
+    }
+
+    public static void readData() {
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Integer> data = new ArrayList<>();
+        while (true) {
```

Comment:
Add any sout here, as after running the program there is nothing in console and it isn't understandable what user have to do.